modules/ebi-metagenomics/easel/eslsfetch/meta.yml

```diff
@@ -0,0 +1,49 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "easel_eslsfetch"
+description: Extract fasta sequences by name from a cmsearchdeoverlap result
+keywords:
+  - fasta
+  - cmsearch
+  - easel
+tools:
+  - "easel":
+      description: "Easel is an ANSI C code library for computational analysis of biological sequences using probabilistic models."
+      homepage: "https://github.com/EddyRivasLab/easel"
+      documentation: "https://github.com/EddyRivasLab/easel"
+      tool_dev_url: "https://github.com/EddyRivasLab/easel"
+      licence: ["BSD"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', single_end:false ]`
+  - fasta:
+      type: file
+      description: Fasta sequence file from amplicon sequencing
+      pattern: "*.{fasta, fasta.gz, fa, fa.gz}"
+  - cmsearch_deoverlap_out:
+      type: file
+      description: Tabular output from 'cmsearchtnloutdeoverlap' module
+      pattern: "*.{tblout.deoverlap}"
```

Comment:
maybe that name should not be hardcoded here `tblout.deoverlap`
probably it is just enough saying it should be tab separated file

Reply:
Agree with Kate, just state the extension.
Same here:
*easel_coords.fasta -> *.fasta

Reply:
> Agree with Kate, just state the extension. Same here: *easel_coords.fasta -> *.fasta

I specify this one because the input is also a fasta file and I don't want it to grab both as output